### PR TITLE
ui-fixes

### LINKED ISSUE #10669: Migrated custom infowindow close button doesn't work
### Context
- Close button doesn't work, because it doesn't contain the necessary class `js-close`.

### Steps to Reproduce
0. Choosing a user with the builder disabled.
1. Create a map in the editor, with a dataset of points, for example.
2. Add things in a string column if needed.
3. Add a popup adding any column to it.
4. Make that popup a custom one, selecting it as HTML, save it.
5. Enable the builder.
6. Go to the map in the builder and open a popup.

### Current Result
It works, but you can't close the popup.
 
### Expected result
We should be able to close it

### Browser and version
Any

### PR DESCRIPTION
**Pull request by piensaenpixel**
Fixes #10669.

@nobuti could you take a look?

Fix notifier in dark view (we were showing a white line and change for this one)

image

Fix hide layer
hide

cc/ @saleiva

### CODE REVIEW

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 1]
```diff
@@ -146,9 +146,12 @@
   margin-top: 16px;
 }
 .Editor-ListAnalysis:empty,
-.Editor-ListAnalysis.is-hidden {
+.Editor-ListAnalysis.is-collapse {
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: is-collapsed?

**Comment by nobuti** [comment]
:99:

**Comment by nobuti** [comment]
💯

**Comment by nobuti** [comment]
I think sequential analysis is not working as expected:

screen shot 2016-11-09 at 20 01 50

cc @saleiva @dgaubert

**Comment by xavijam** [comment]
Ready to go then?

**Comment by piensaenpixel** [comment]
\o/

**Pull request merged**